docker-compose.yml

```diff
@@ -87,6 +90,8 @@ services:
       - network
     depends_on:
       - backend
+    ports:
+      - "${FE_PORT:-4200}:4200"
```

Comment:
🔴 I want to raise one issue here.

in the past we had port bindings in the `docker-compose.yml`. We removed them because exposed ports cannot be closed by writing anything in the `docker-compose.override.yml`. Therefore, we considered the practice of not exposing ANY port in the `docker-compose.yml`.

This led to having the `docker-compose.override.example.yml` to contain all port bindings. Hence, I'd remove that port bindings from here.

It is very easy to understand, that a port is not acessible from the host, and then a dev can use the `docker-compose.override.example.yml` to get the "default" port bindings.

Prematurely bound ports is always a hassle, when you want to up stacks and a port is not available.

Reply:
Can surely be removed, though there is a solution for overriding using yaml tags: [!reset](https://docs.docker.com/reference/compose-file/merge/#reset-value) or [!override](https://docs.docker.com/reference/compose-file/merge/#replace-value)

Reply:
@toy In that case it can be removed, yes. But please update the override example accordingly to use the `FE_PORT` in the frontend section.

Also we should amend the docker development readme to point out that this override file is necessary to be able to access the setup under HTTP://localhost:3000 as documented.

Reply:
Ah right! All good then!

Reply:
That is this line can be removed.